cpp/include/Ice/EndpointSelectionType.h

```diff
@@ -9,6 +9,7 @@ namespace Ice
 {
     /**
      * Determines the order in which the Ice run time uses the endpoints in a proxy when establishing a connection.
+     * \headerfile Ice/Ice.h
```

Comment:
This is ok but not useful as doxygen does not generate a separate page for enum types.